node/src/main/scala/org/bitcoins/node/Node.scala

```diff
@@ -156,10 +156,9 @@ trait Node extends NodeApi with ChainQueryApi with P2PLogger {
   def stop(): Future[Node] = {
     logger.info(s"Stopping node")
     val disconnectF = for {
-      _ <- nodeAppConfig.stop()
-      _ <- chainAppConfig.stop()
       p <- peerMsgSenderF
       disconnect <- p.disconnect()
+      _ <- nodeAppConfig.stop()
```

Comment:
Is there a reason we aren't stopping the chain?

Reply:
So IIRC this was causing issues in test cases because shutting down the node doesn't necessarily mean you want to shutdown the chain project. 

The node is _dependent_ upon the chain, but the chain _isn't_ dependent upon on the node. So therefore the node shouldn't be able to stop the chain project from operating.

In the case of `appSever` that may be true, but `wallet` also uses the chain project. I don't think we want to the chain to stop functioning necessarily just because we want to shutdown the node. They are independent projects.

Therefore the caller of the `node.stop()` method should make the conscious choice to shutdown chain as well, rather than the node doing it on it's own.